src/ec/packages/ec-component-tag/tag.story.js

```diff
@@ -1,63 +1,45 @@
-import merge from 'deepmerge';
+/* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
-
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import dataLink from './demo/data--link';
 import dataButton from './demo/data--button';
 import dataRemovable from './demo/data--removable';
 import tag from './ecl-tag.html.twig';
 import notes from './README.md';
 
+// Preserve the adapted specs.
+const prepareTag = data => {
+  data.default_icon_path = defaultSprite;
+  data.tag.label = text('label', data.tag.label, buttonLabels.required);
```

Comment:
but if it's
`data.tag.label` the label is expect is:
`tag.label`
You have t think that this is the property in the input data